### DISCUSSION TITLE
Limit of requests in NetBox

### DISCUSSION THREAD

**Discussion by athira009** [netbox-community/netbox#14189] [Q&A]
Could you please provide the limit of requests/month?
Could you please provide the limit for the number of devices/IPs

**Comment by candlerb** [comment]
See #14180.

What makes you think Netbox would have such limits - and if it did, why it would not document them?

**Reply by athira009** [reply]
Hi @candlerb , https://github.com/netbox-community/netbox/discussions/14180. was about daily limits. Limit is mentioned in the pricing section of NetBox cloud. So, I doubt if there is any limit for open source NetBox.

**Reply by jcollie** [reply]
The open source version of NetBox has no limits, except the limits of your hardware. I thought that there was some basic rate limiting for API calls to prevent simple DDOS attacks but I can't find any documentation to back that up. Maybe I'm thinking of some other project. Questions about the rate limits of the hosted cloud version are best directed to the NetBox Labs people.
